new/LIT6180HomilyChurches.xml

```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6180HomilyChurches" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ድርሳን፡ በእንተ፡ ሕንጻ፡ አብያተ፡ ክርስቲያን፡ ለእግዝእትነ፡ ቅድስት፡ ማርያም፡ በዕለተ፡ ቅዳሴሃ፡ አመ፡ ፳ወ፩ለሰኔ፡</title>
+                <title xml:lang="en" xml:id="t2">Homily on the construction of churches dedicated to Mary for the day of her
+                feast on 21st of Sane</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The title of the work has been taken from EMML 8897, <locus target="#49r">f. 49r</locus>.</p>
+            </abstract>
+
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Homily"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MKr" when="2020-09-15">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="ecrm:P129_is_about" active="LIT6180HomilyChurches" passive="PRS6819Mary"/>
+                    <relation name="saws:isDifferentTo" active="LIT6180HomilyChurches" passive="LIT6178BasilPhilippi"/>
```

Comment:
The attribution to an author can also be encoded here (saws:isAttributedToAuthor), with a precise description and a ref to the Ms

Reply:
Thank you for that. In EMML 8897 it is attributed to John of Cesarea, my intuition is that they made a mistake and it should be Basil of C., but it would take some research to confirm. So for the time being I ignored the attribution.